packages/volto-dsgvo-banner/src/components/IfConfirm/IfConfirm.jsx

```diff
@@ -36,6 +46,36 @@ const IfConfirm = ({ children, module, showDSGVOBanner }) => {
           </p>
         </Message>
       </div>
+    ) : (
+      <div className="dsgvo-overlay-wrapper">
+        {!isConfirmed && (
+          <div className="dsgvo-overlay">
+            <div className="dsgvo-overlay-content">
+              <p>
+                <FormattedMessage
+                  id="This content is provided by a third-party service. To view it, please allow cookies for {module} in your privacy settings."
+                  defaultMessage="This content is provided by a third-party service. To view it, please allow cookies for {module} in your privacy settings."
+                  values={{
+                    module: <b>{module}</b>,
+                  }}
+                />
+              </p>
+              <div>
+                <Button
+                  className="dsgvo-allow-btn"
+                  onClick={() => confirmModuleOverlay(module)}
+                >
+                  <FormattedMessage
+                    id="Allow and Continue"
+                    defaultMessage="Allow and Continue"
+                  />
+                </Button>
+              </div>
+            </div>
+          </div>
+        )}
+        <div>{children}</div>
```

Comment:
@Tishasoumya-02 We can't actually render the children until the user has accepted the cookies. Rendering the children will load the third-party content which sets cookies, and we aren't allowed to do that until we get the user's consent. We have to show a placeholder instead.